go/enclave/enclave_test.go

```diff
@@ -229,7 +229,7 @@ func gasEstimateInvalidNumParams(t *testing.T, w wallet.Wallet, enclave common.E
 
 	// Run gas Estimation
 	_, err = enclave.EstimateGas(encryptedParams)
-	if !assert.ErrorContains(t, err, "required at least one param") {
+	if !assert.ErrorContains(t, err, " required exactly two params, but received 0") {
```

Comment:
Eth call *must* be 2 args - https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_call

```
curl --location --request POST 'http://testnet-gethnetwork.uksouth.azurecontainer.io:8025/' \
--header 'Content-Type: application/json' \
--data-raw '{
    "jsonrpc":"2.0",
    "method":"eth_getTransactionCount",
    "params":[
        "0x6c78ce7b104f56f5339b2e76b22a314a2e7d1794"
    ],
    "id":1
}'
```
vs 
```
curl --location --request POST 'http://testnet-gethnetwork.uksouth.azurecontainer.io:8025/' \
--header 'Content-Type: application/json' \
--data-raw '{
    "jsonrpc":"2.0",
    "method":"eth_getTransactionCount",
    "params":[
        "0x6c78ce7b104f56f5339b2e76b22a314a2e7d1794",
        "latest"
    ],
    "id":1
}'
```

Reply:
After some more digging:

`For some Ethereum clients/simulators the [default block parameter](https://eth.wiki/json-rpc/API#the-default-block-parameter) for eth_call is optional, but for Go Ethereum it is not.`

Reverting this 👍